README.md

```diff
@@ -834,6 +834,8 @@ Differences from GoSite:
 
 `api/profiles/Advisors/:username` Get advisor(s) info of a user with username `username` as a parameter.
 
+`api/profiles/clifton/:username` Get the Clifton Strengths of a user with username `username` as a parameter. The returned object includes all columns in the db table, which are currently `id_num`, `access_code`, `email`, `dte_completed`, `theme_1`, `theme_2`, `theme_3`, `theme_4`, `theme_5`.
```

Comment:
I don't think we're in the habit of documenting what info is returned in detail, are we? It seems like we should probably defer to the rest of the documentation here and let the code document itself. Once again, a ViewModel would help clarify exactly what data is being accessed, without having to duplicate that info in the README and risk them being out of sync.

Reply:
Specifically, I think returning ID_Num from this publicly accessible route is a bad idea. But we should be in the habit of only serving the minimum information that is useful.